static/js/recent_topics_ui.js

```diff
@@ -442,6 +438,11 @@ function format_conversation(conversation_data) {
             }
         }
 
+        // Display in most recent sender first order.
+        // TODO: We only display the participants of PM conversation right now, but in future we may want
+        // to show non-participants only for PMs, perhaps in a different shade.
```

Comment:
Proposed comment:
```
// To match the behavior for streams, we display the set of users who've actually
// participated, with the most recent participants first. It could make sense to 
// display the other recipients on the PM conversation with different styling,
// but it's important to not destroy the information of "who's actually talked".
```